frame/bonded-finance/src/tests.rs

```diff
@@ -179,6 +179,42 @@ prop_compose! {
 			  }
 }
 
+#[test]
+fn bond_offer_cc_e005bce405358673d3249e53d51e474d4b135bdf8f2b0b82f94f525c0efbdbf8() {
```

Comment:
If I remember correctly, proptest will automatically replay the tests saved in the `proptest-regressions` directory, we should create a unit test if the test was a corner case, but in this case it's just precision (not really something different in the logic)

Reply:
that is weird. i created file by rules. but it was not triggered. strange.

Reply:
Weird because it's stated in their doc https://altsysrq.github.io/proptest-book/proptest/failure-persistence.html